src/app/researcher/study/[studyId]/review/page.tsx

```diff
@@ -47,19 +47,43 @@ export default async function StudyReviewPage(props: { params: Promise<{ studyId
 
             <Paper bg="white" p="xxl">
                 <Stack>
-                    <Title order={4} size="xl">
-                        Study Code
-                    </Title>
+                    <Group justify="space-between" align="center">
+                        <Title order={4} size="xl">
+                            Study Code
+                        </Title>
+                        <StudyStatusDisplay
+                            status={
+                                job?.latestStatus === 'CODE-APPROVED'
+                                    ? 'APPROVED'
+                                    : job?.latestStatus === 'CODE-REJECTED'
+                                      ? 'REJECTED'
+                                      : null
+                            }
+                            date={job?.latestStatusChangeOccurredAt}
+                        />
+                    </Group>
                     <Divider c="dimmed" />
                     <StudyCodeDetails job={job} />
                 </Stack>
             </Paper>
 
             <Paper bg="white" p="xxl">
                 <Stack>
-                    <Title order={4} size="xl">
-                        Study Results
-                    </Title>
+                    <Group justify="space-between" align="center">
+                        <Title order={4} size="xl">
+                            Study Results
+                        </Title>
+                        <StudyStatusDisplay
+                            status={
+                                job?.latestStatus === 'RESULTS-APPROVED'
+                                    ? 'APPROVED'
+                                    : job?.latestStatus === 'RESULTS-REJECTED'
+                                      ? 'REJECTED'
+                                      : null
+                            }
+                            date={job?.latestStatusChangeOccurredAt}
+                        />
```

Comment:
maybe lets add a `StudyStatusApprovalStatus` component or something like that in components/study/display-study-status and use that for both of these spots?